infra/charts/feast-feature-server/templates/deployment.yaml

```diff
@@ -14,6 +14,9 @@ spec:
     {{- with .Values.podAnnotations }}
       annotations:
         {{- toYaml . | nindent 8 }}
+        {{- if .Values.metrics }}
```

Comment:
this should probably include the entire `annotations` block

Reply:
sorry, i think i was wrong about that. it appears `annotations` was preexisting. let's revert my ask